ui/screens/StreamListRow.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { ScrollView, StyleSheet, Text, View } from 'react-native';
+import { Avatar, Card , Input, ListItem, Icon, Button, Header} from "react-native-elements";
+
+
+
+const list = [
+  {
+    pk: 2,
+    title: 'Craze of Club House',
+    description: 'Recently People are shifting to clubhouse',
+    followed_by: 400,
+  },
+  {
+    pk: 2,
+    title: 'Craze of Club House',
+    description: 'Recently People are shifting to clubhouse',
+    followed_by: 400,
+  },
+  {
+    pk: 2,
+    title: 'Craze of Club House',
+    description: 'Recently People are shifting to clubhouse',
+    followed_by: 400,
+  },
+  {
+    pk: 2,
+    title: 'Craze of Club House',
+    description: 'Recently People are shifting to clubhouse',
+    followed_by: 400,
+  },
+  {
+    pk: 2,
+    title: 'Craze of Club House',
+    description: 'Recently People are shifting to clubhouse',
+    followed_by: 400,
+  },
+  {
+    pk: 2,
+    title: 'Craze of Club House',
+    description: 'Recently People are shifting to clubhouse',
+    followed_by: 400,
+  },
+
+];
+
+
+export default function App() {
```

Comment:
use arrow functions like const StreamListRow = () => {}  and then export it so it can be used.